README.md

```diff
@@ -181,9 +183,44 @@ The customization of the model can also be provided, with the prop parts:
 
 ![Image with Parts Example](res/images/image-parts.png)
 
-There can be more than one image using the same instance of Ripe SDK:
+The Image component support more complex logic and attributes, such as different formats, crops and personalization options.
+By setting the `show-initials` to `true` and providing an `initials-group` and `state`, it is possible to show initials in the model.
+
+A `state` example:
 
+```javascript
+const state = {
+    initialsExtra: {
+        main: {
+            initials: "A",
+            engraving: "style:white"
+        }
+    }
+};
 ```
+
+```html
+<ripe-image
+    v-bind:brand="'dummy'"
+    v-bind:model="'cube'"
+    v-bind:version="52"
+    v-bind:size="1000"
+    v-bind:frame="'side-0'"
+    v-bind:format="'png'"
+    v-bind:crop="true"
+    v-bind:show-initials="true"
+    v-bind:initials-group="'main'"
+    v-bind:state="state"
+/>
+```
+
+![Image with Personalization](res/images/image-personalization.png)
+
+It is also possible to provide a `initialsBuilder` function, that allows for a more custom logic in the translation from initials and engraving into initials and profiles.
+
+There can be more than one image using the same instance of Ripe SDK:
```

Comment:
Maybe this should be under a different section, right now it's inside `Image` which I don't think it's what you intended right?

Reply:
@gcandal The part with the multiple SDKs, or the one related to the `initialsBuilder`?

Reply:
Both of them are related to the image